maestro-ios/src/main/java/ios/xctest/XCTestIOSDevice.kt

```diff
@@ -91,7 +91,9 @@ class XCTestIOSDevice(
     }
 
     override fun pressKey(code: Int): Result<Unit, Throwable> {
-        error("Not supported")
+        return runCatching {
+            client.inputText("\n")
+        }
```

Comment:
Should return an error if code != 40?

Reply:
Thanks for the suggestion, that would be better.